<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeModifierDialogWidget</class>
 <widget class="QWidget" name="RangeModifierDialogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>175</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,10">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string comment="start of the range">From:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KIntNumInput" name="startInput" native="true">
       <property name="minimum" stdset="0">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QCheckBox" name="toTheEndCheckBox">
       <property name="text">
        <string comment="range goes to the end of the string">to the end</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string comment="end of the range">To:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KIntNumInput" name="stopInput" native="true">
       <property name="minimum" stdset="0">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
